gen_training_0.25d.py

```diff
@@ -240,8 +260,20 @@ def era5_process(self):
     output_directory = args.output
     download_directory = args.download
     keep_downloaded_data = args.keep
-
-    data_processor = DataProcessor(start_date, end_date, output_directory, download_directory, keep_downloaded_data)      
+    
+    gfs_variables_with_levels = {
+        '2 metre temperature': [{'typeOfLevel': 'heightAboveGround', 'level': 2}],
+        'Temperature': [{'typeOfLevel': 'isobaricInhPa', 'level': 1000},{'typeOfLevel': 'surface', 'level': 0}],
+        'Relative humidity': [{'typeOfLevel': 'isobaricInhPa', 'level': 1000}],
+        'Pressure reduced to MSL': [{'typeOfLevel': 'meanSea', 'level': 0}],
+        '10 metre U wind component': [{'typeOfLevel': 'heightAboveGround', 'level': 10}],
+        '10 metre V wind component': [{'typeOfLevel': 'heightAboveGround', 'level': 10}],
+        'Soil Temperature': [{'typeOfLevel': 'depthBelowLandLayer', 'level': 0}],
```

Comment:
Can you double check this , usually we have this "0-0.1 m"
827:50829985:d=2021032206:TSOIL:0-0.1 m below ground:24 hour fcst:

Reply:
this gives me "564:Soil temperature:K (instant):regular_ll:depthBelowLandLayer:levels 0.0-0.1 m:fcst time 0 hrs:from 202203030000" 
is it the wrong one?

Reply:
OK, this is the correct one. Thanks for confirm.

Reply:
I'm loading them by the keys using the pygrib libraries, its a little bit different from wgrib2. Also, when I open a grib2 file using pygrib we have 696 records, idk why yours showing #827. Thanks

Reply:
I was looking at a GFSv7 regression test grib file, which is different from the GSFv16 opn files.